tests/unit_tests/dagmc/test_lost_particles.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+from pathlib import Path
+
+import openmc
+import openmc.lib
+
+import pytest
+
+pytestmark = pytest.mark.skipif(
+    not openmc.lib._dagmc_enabled(),
+    reason="DAGMC CAD geometry is not enabled.")
+
+
+@pytest.fixture
+def broken_dagmc_model(request):
+    openmc.reset_auto_ids()
+    model = openmc.model.Model()
+
+    ### MATERIALS ###
+    fuel = openmc.Material(name='no-void fuel')
+    fuel.set_density('g/cc', 10.29769)
+    fuel.add_nuclide('U234', 0.93120485)
+    fuel.add_nuclide('U235', 0.00055815)
+    fuel.add_nuclide('U238', 0.022408)
+    fuel.add_nuclide('O16', 0.045829)
+
+    cladding = openmc.Material(name='clad')
+    cladding.set_density('g/cc', 6.55)
+    cladding.add_nuclide('Zr90', 0.021827)
+    cladding.add_nuclide('Zr91', 0.00476)
+    cladding.add_nuclide('Zr92', 0.0072758)
+    cladding.add_nuclide('Zr94', 0.0073734)
+    cladding.add_nuclide('Zr96', 0.0011879)
+
+    water = openmc.Material(name='water')
+    water.set_density('g/cc', 0.740582)
+    water.add_nuclide('H1', 0.049457)
+    water.add_nuclide('O16', 0.024672)
+    water.add_nuclide('B10', 8.0042e-06)
+    water.add_nuclide('B11', 3.2218e-05)
```

Comment:
Can you cut these down to a single nuclide each? It's not important for the sake of the test and results in more time in the test just loading cross sections